pagopa/io-app
# chore(Piattaforma Notifiche): [IAMVL-90] Use PN remote config

## Short description
This PR uses [the remote feature flag](https://github.com/pagopa/io-services-metadata/pull/639) (chained with the local one) in order to enable/disable the PN feature.

## List of changes proposed in this pull request
- Bump `io-services-metadata` version
- Add two new selectors to get the config (it will be used for the `frontend_url`) and the flag value
- Use the new selector (instead of the local FF) throughout the app

## How to test
Use the [IIP-18-add-pn-endpoints](https://github.com/pagopa/io-dev-api-server/tree/IIP-18-add-pn-endpoints) branch of the dev server and add this to your `config.json`:
```
{
  "messages": {
    "pnCount": 1
  },
  "services": {
    "includePn": true
  }
}
```
Make sure you set `PN_ENABLED=YES` to `.env` and run the app. By changing the `BackendStatus` in `src/payloads/backend.ts` of the dev-server you should be able to enable/disable the feature in-app.

### Service
| FF Disabled | FF Enabled | 
| - | - |
| ![Simulator Screen Shot - iPhone 13 - 2022-07-26 at 17 09 42](https://user-images.githubusercontent.com/467098/181043092-706f2d8a-162e-4315-8e29-d6141f96bb42.png) | ![Simulator Screen Shot - iPhone 13 - 2022-07-26 at 17 08 46](https://user-images.githubusercontent.com/467098/181043161-57e4269d-d33f-4a85-95ac-a0a5fb0d8b50.png) |

### Message Details
| FF Disabled | FF Enabled | 
| - | - |
| ![Simulator Screen Shot - iPhone 13 - 2022-07-26 at 17 10 00](https://user-images.githubusercontent.com/467098/181043276-0d29e810-4b22-4d8f-8d0d-f768e3d36288.png) | ![Simulator Screen Shot - iPhone 13 - 2022-07-26 at 17 10 41](https://user-images.githubusercontent.com/467098/181043315-24716179-b41d-42d8-b6bd-a9ff22c945c3.png) |
